### PR TITLE
[NPM] add npm-tools k8s workload

### PR DESCRIPTION
**Pull request by nplanel**
What does this PR do?
---------------------

Add npm-tools k8s workload

Which scenarios this will impact?
-------------------

Motivation
----------

Additional Notes
----------------

### CODE REVIEW

**Comment by pducolin** [comment]
If you need it deployed to the Github registry, you need to update the github workflow, I listed raw instructions at https://datadoghq.atlassian.net/wiki/spaces/ADX/pages/3536257272/How+to+add+a+workload+app+for+tests

**Comment by nplanel** [comment]
> If you need it deployed to the Github registry, you need to update the github workflow, I listed raw instructions at https://datadoghq.atlassian.net/wiki/spaces/ADX/pages/3536257272/How+to+add+a+workload+app+for+tests

Thanks but I would not need to update the image, it's just the k8s job description

File: components/datadog/apps/npm-tools/k8s.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+								pulumi.String("-c"),
+								pulumi.String(fmt.Sprintf("while [ 1 ] ; do curl %s ; dig @8.8.8.8 www.google.ch ; sleep 20 ; done", testURL)),
+							},
+							Resources: &corev1.ResourceRequirementsArgs{
+								Limits: pulumi.StringMap{
+									"cpu":    pulumi.String("200m"),
+									"memory": pulumi.String("64Mi"),
+								},
+								Requests: pulumi.StringMap{
+									"cpu":    pulumi.String("200m"),
```
Comment (vboulineau) [inline components/datadog/apps/npm-tools/k8s.go:75]: Seems big for what it does. `50m` should be enough

**Pull request merged**